fiat-roles/src/test/groovy/com/netflix/spinnaker/fiat/providers/ResourcePrefixPermissionSourceSpec.groovy

```diff
@@ -44,9 +44,9 @@ class ResourcePrefixPermissionSourceSpec extends Specification {
 
         where:
         applicationName     | expectedCreatePermissions
-        'new york'          | ["admins"]
-        'gotham-criminals'  | ["admins", "police"]
-        'gotham-joker'      | ["admins", "police", "batman"]
+        'NEW YORK'          | ["admins"]
+        'GOTHAM-CRIMINALS'  | ["admins", "police"]
```

Comment:
can you update these test cases to include a mix of upper and lower case (it looks like your changes should ensure that works)

Reply:
Added the testcases.

Reply:
Added the testcases.